pyproject.toml

```diff
@@ -319,17 +319,21 @@ ignore = [
   "TD002",   # We don't have require authors in TODO
   "TD003",   # We don't have an issue link for all TODOs today
   "FIX002",  # Line contains TODO - ignoring for ruff for now
+  "TRY004",  # Because of magics for nice exception message
   "F821",    # Disable undefined-name until resolution of #10451
 ]
 
-
 # Allow autofix for all enabled rules (when `--fix`) is provided.
 fixable = ["ALL"]
 unfixable = []
 
 # Allow unused variables when underscore-prefixed.
 dummy-variable-rgx = "^(_+|(_+[a-zA-Z0-9_]*[a-zA-Z0-9]+?))$"
 
+[tool.ruff.lint.pylint]
+max-args = 7
+max-branches = 13
+
```

Comment:
do we need this in addition to pylintrc?

Reply:
Yeah, they do not even catch the same errors...

Reply:
are they both aligned?